extensions/compose/package.json

```diff
@@ -19,7 +20,35 @@
         "command": "compose.checks",
         "title": "Compose: Checks..."
       }
-    ]
+    ],
+    "onboarding": {
+      "title": "Compose Setup",
+      "steps": [
+        {
+          "id": "composeSettings",
+          "label": "Compose Settings",
+          "title": "Compose Settings",
+          "content": [
+            [
+              {
+                "value": "${configuration:compose.binary.installComposeSystemWide}"
+              }
+            ]
+          ]
+        }
+      ]
+    },
+    "configuration": {
+      "title": "Compose",
+      "properties": {
+        "compose.binary.installComposeSystemWide": {
+          "type": "boolean",
+          "default": false,
+          "scope": ["DEFAULT", "Onboarding"],
```

Comment:
Encountered issue https://github.com/containers/podman-desktop/issues/3977 so this part need to be removed before merging.